fastmri_recon/data/test_generators.py

```diff
@@ -38,8 +41,53 @@ def flow_random_shapes(self,):
             mask_batch = mask_batch[..., 0]
             yield (kspaces, mask_batch), images
 
+    @threadsafe_generator
     def flow_z_filled_random_shapes(self,):
         random_shapes_gen = self.flow_random_shapes()
-        for (kspaces, _), images in random_shapes_gen:
+        for (kspaces, mask_batch), images in random_shapes_gen:
+            z_filled = zero_filled_recon(kspaces[..., 0], crop=False)[..., None]
+            yield z_filled, images, mask_batch
```

Comment:
why return the mask too now?

Reply:
It was for testing purposes, my bad for leaving it there.